tailwind.config.js

```diff
@@ -3,7 +3,19 @@ export default {
   mode: "jit",
   content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        montaga: ["montaga"],
+        montserrat: ["Montserrat"],
+        nats: ["nats"],
+      },
+      colors: {
+        linkFill: "#FAFAFA",
+        linkBorder: "#999494",
+        navColor: "#D9D9D9",
+      },
+    },
   },
+  extend: {},
```

Comment:
any reason to keep this extend as well as the one on line 6?

Reply:
it was already in the tailwind config, not sure if it can be used to extend in a different way